core/embed/rust/src/ui/event.rs

```diff
@@ -51,6 +51,8 @@ pub enum TouchEvent {
     TouchMove(Point),
     /// Touch has ended at a point on the screen.
     TouchEnd(Point),
+    /// Touch has been aborted.
```

Comment:
i don't understand what this does

Reply:
tried to make it clearer https://github.com/trezor/trezor-firmware/commit/52193a77188a1f7f3905ce948311f70085bf4578